### ISSUE TITLE
seemingly mistaken "unsupported platform" error during `npm install` step

### ISSUE THREAD

**Issue by wgwz** [thenewinquiry/bailbloc#51]
👋🏻 Hi, everyone. New here, looking to help out. I'm trying to get things running from the repo locally and I ran into this during the `npm install` step. It seems like I'm getting the wrong dependency, `7zip-bin-linux`. 

```
$ npm install
npm WARN bailbloc@1.1.3 No repository field.

npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for 7zip-bin-linux@1.2.0: wanted {"os":"linux","arch":"any"} (current: {"os":"darwin","arch":"x64"})
npm ERR! notsup Valid OS:    linux
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   darwin
npm ERR! notsup Actual Arch: x64

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/wizdre/.npm/_logs/2018-01-21T19_16_48_807Z-debug.log
```

Thinking maybe this is a non-issue, when I go to next step:

```
$ npm start

> bailbloc@1.1.3 start /Users/wizdre/code/bailbloc/desktop
> electron .

sh: electron: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! bailbloc@1.1.3 start: `electron .`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the bailbloc@1.1.3 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/wizdre/.npm/_logs/2018-01-21T19_35_28_029Z-debug.log
```
Any help appreciated, glad to add to the docs once I figure out how to fix this.

**Comment by wgwz** [comment]
Looks like it may be this line: https://github.com/wgwz/bailbloc/commit/a8b30c7ef0e1cc10742831b50c9bb2cc54653fea#diff-939672cf2fde9f005fd23d0b587deec1R21

**Comment by wgwz** [comment]
[potentially related](https://github.com/electron-userland/electron-builder/issues/2013)

**Comment by wgwz** [comment]
Fixed w #47
